src/components/GeriLifeTopbar.vue

```diff
@@ -152,9 +166,20 @@ export default {
       }
     },
 
+    clicked(menuItem) {
+      this.selected = menuItem.title;
+      if (menuItem.isProfileLink) {
+        this.openProfile = true;
+      } else {
+        this.$router.push({ path: menuItem.route });
+      }
+    },
```

Comment:
This function is probably redundant if we use router links for the menu items.

Go ahead and separate the profile link/button from the other navigation links, since it isn't behaving the same.

Reply:
done